### PR TITLE
Update image and README

### PR DESCRIPTION
**Pull request by rbehjati**
Fixes #<issue_number_goes_here>

> It's a good idea to open an issue first for discussion.

- [ ] Tests pass
- [ ] Appropriate changes to README are included in PR

### CODE REVIEW

**Comment by rbehjati** [comment]
@mariaschett Can you have a quick look at this? This is mostly formatting, but I've added a new subsection in the README for _Related work and standards_. It is not complete, but I thought it could be a good start.

File: docs/claim-transparency.md [hunk 1]
```diff
@@ -320,3 +317,33 @@ shown in the following example.
...
+The Remote ATtestation procedureS (RATS) working group has provided an
+[architecture](https://datatracker.ietf.org/doc/html/draft-ietf-rats-architecture) and glossary of
+concepts related to remote attestation.
+[This cheatsheet](https://github.com/thomas-fossati/rats-cheatsheet) and
+[this slides deck](https://confidentialcomputing.io/wp-content/uploads/sites/85/2021/09/IETF-Remote-Attestation-Architecture-Overview.pdf)
+give an overview of the architecture and the main concepts. RATS has many concepts similar to the
+ones in our design, but seems to be focused on claims and evidence that are generated and consumed
+automatically. Claims and evidence in RATS are designed to be used for remote attestation. The
+claims in our binary transparency ecosystem, however, are not limited to the ones used for remote
+attestation. We target a wider range of use cases.
```
Comment (mariaschett) [inline docs/claim-transparency.md:339]: Could be nice to give an example of a use case we target which RATS does not.

Comment (rbehjati) [reply docs/claim-transparency.md]: Thanks. Done.

File: docs/claim-transparency.md [hunk 2]
```diff
@@ -320,3 +317,33 @@ shown in the following example.
...
+
+In addition, the concept of
+[Concise Reference Integrity Manifest (CoRIM)](https://datatracker.ietf.org/doc/draft-ietf-rats-corim/)
+provided by the RATS working group is very similar to a `Claim` as described in this document.
+
+TODO: Summarize the comparison.
```
[the remark this answers is not in the document]
Comment (rbehjati) [reply docs/claim-transparency.md]: Done.

File: docs/claim-transparency.md [hunk 3]
```diff
@@ -320,3 +317,33 @@ shown in the following example.
...
+
+### Comparison to SCITT
+
+TBA.
```
[the remark this answers is not in the document]
Comment (rbehjati) [reply docs/claim-transparency.md]: Done.

**Review by rbehjati — COMMENTED** [review]
Thanks for the review :)

**Pull request merged**